tracer/test/benchmarks/Benchmarks.Trace/Program.cs

```diff
@@ -67,6 +78,52 @@ private static void Main(string[] args)
 
             Console.WriteLine("Running tests...");
             BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args, config);
+
+            return 0;
```

Comment:
Did we check if `BenchmarkSwitcher.Run` changes the `Environment.ExitCode`? Maybe to play safe we can do `return Environment.ExitCode;` here

Reply:
Fixed in 1b1c7ab60d5b54ff40541d542f5d2c1c516108c3